deepsampler-core/src/main/java/de/ppi/deepsampler/core/api/Execution.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020  PPI AG (Hamburg, Germany)
+ * This program is made available under the terms of the MIT License.
+ */
+
+package de.ppi.deepsampler.core.api;
+
+import de.ppi.deepsampler.core.model.ExecutionRepository;
+import de.ppi.deepsampler.core.model.SampleRepository;
+
+/**
+ * Provides functionality for influencing the execution phase of the stubbing done by deepsampler.
```

Comment:
How do we write deepsampler? DeepSampler od deep-sampler or DeepSampler? I think we use all three versions... maybe we should unify that...